cmake/SndFileConfig.cmake.in

```diff
@@ -9,10 +9,12 @@ set (SndFile_WITH_EXTERNAL_LIBS @SndFile_WITH_EXTERNAL_LIBS@)
 
 include (CMakeFindDependencyMacro)
 
-find_dependency (Ogg 1.3)
-find_dependency (VorbisEnc)
-find_dependency (FLAC)
-find_dependency (Opus)
+if (SndFile_WITH_EXTERNAL_LIBS)
+	find_dependency (Ogg 1.3)
+	find_dependency (VorbisEnc)
```

Comment:
Why not add `REQUIRED` to all `find_dependency`? It is safe now.

Reply:
Adding `REQUIRED` here would make CMake configuration fail for projects that call `find_package(SndFile)`: libsndfile isn't required, so libogg, etc. shouldn't be required.

However, if a project calls `find_package(SndFile REQUIRED)`, then [`find_dependency()`](https://cmake.org/cmake/help/v3.17/module/CMakeFindDependencyMacro.html) forwards the `REQUIRED` argument when it calls `find_package()` internally.